lib/infra/infra-stack.ts

```diff
@@ -611,20 +612,30 @@ export class InfraStack extends Stack {
 
       if (props.securityDisabled && !props.minDistribution) {
         cfnInitConfig.push(InitCommand.shellCommand('set -ex;cd opensearch-dashboards;'
-            + './bin/opensearch-dashboards-plugin remove securityDashboards --allow-root;'
-            + 'sed -i /^opensearch_security/d config/opensearch_dashboards.yml;'
-            + 'sed -i \'s/https/http/\' config/opensearch_dashboards.yml',
+          + './bin/opensearch-dashboards-plugin remove securityDashboards --allow-root;'
+          + 'sed -i /^opensearch_security/d config/opensearch_dashboards.yml;'
+          + 'sed -i \'s/https/http/\' config/opensearch_dashboards.yml',
         {
           cwd: '/home/ec2-user',
           ignoreErrors: false,
         }));
       }
 
       cfnInitConfig.push(InitCommand.shellCommand('set -ex;cd opensearch-dashboards;'
-          + 'sudo -u ec2-user nohup ./bin/opensearch-dashboards > dashboard_install.log 2>&1 &', {
+        + 'sudo -u ec2-user nohup ./bin/opensearch-dashboards > dashboard_install.log 2>&1 &', {
         cwd: '/home/ec2-user',
         ignoreErrors: false,
       }));
+
+      // @ts-ignore
```

Comment:
This setting should be added before dashboards are started else they will not be picked up.